common-lib/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 0.3.0
+
+- [Signal] add new signal `log`. add enableLokiLogs=true|false to signals init
```

Comment:
Might be worth splitting this on a multi-line entry, and making `enableLokiLogs` a code block

Reply:
updated